geometry/include/DD4hepGeometry.hh

```diff
@@ -27,9 +27,9 @@ namespace aidaTT
     
 
   private:
-    const dd4hep::Detector& _thedetector ;
+    const dd4hep::Detector& _thedetector{} ;
```

Comment:
This does not work:
```cpp
error: allocating an object of abstract class type 'const dd4hep::Detector'
    const dd4hep::Detector& _thedetector{} ;
```

Reply:
Yes, indeed this this could not work if ever used. However three (!) compilers happily compiled it w/o warning. Only llvm on MacOS actually fails...
Fixed in #21.